composites/OnboardingWizard/onboarding-wizard.scss

```diff
@@ -44,6 +44,23 @@ body {
   box-sizing: border-box;
   width: 100%;
 
+  &--header {
+    text-align: center;
+
+    h1 {
+      color: $color_pink_dark;
+      margin: 0 0 -16px;
+      font-size: 1.2em;
```

Comment:
this translates to a computed value of 19.2px, ideally we should use a value that computes to a number without decimals